kotlin/scene-layer/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.esri.arcgisruntime.sample.scenelayer">
+    <uses-permission android:name="android.permission.INTERNET" />
+    <application
```

Comment:
Let's add OpenGL requirements like here: https://github.com/Esri/arcgis-runtime-samples-android/blob/master/kotlin/view-point-cloud-data-offline/src/main/AndroidManifest.xml#L7

Reply:
sorry i thought this was in there.